addons/parachute/functions/fnc_doLanding.sqf

```diff
@@ -15,9 +15,10 @@
  */
 #include "script_component.hpp"
 private ["_unit"];
```

Comment:
If you use the `PARAMS` macro I don't think you have to wrap it into a private array anymore.